### ISSUE TITLE
Can't install on OSX

### ISSUE THREAD

**Issue by chrisfosterelli** [requiresafe/cli#13]
Any ideas how to resolve this error?
Installing globally without `sudo` fails as well.

```bash
➜  sms git:(inline-style-fix) sudo npm install -g requiresafe
 
> git-validate@2.1.0 install /usr/local/lib/node_modules/requiresafe/node_modules/git-validate
> node bin/install

fs.js:500
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: EACCES, permission denied '/usr/local/.git/hooks/pre-commit'
    at Error (native)
    at Object.fs.openSync (fs.js:500:18)
    at Object.fs.writeFileSync (fs.js:1099:15)
    at Object.exports.installHooks (/usr/local/lib/node_modules/requiresafe/node_modules/git-validate/lib/utils.js:248:12)
    at Object.<anonymous> (/usr/local/lib/node_modules/requiresafe/node_modules/git-validate/bin/install.js:3:7)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "requiresafe"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code ELIFECYCLE

npm ERR! git-validate@2.1.0 install: `node bin/install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the git-validate@2.1.0 install script 'node bin/install'.
npm ERR! This is most likely a problem with the git-validate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node bin/install
npm ERR! You can get their info via:
npm ERR!     npm owner ls git-validate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/chrisfosterelli/workspace/memoryleaf/sms/npm-debug.log
➜  sms git:(inline-style-fix) 
```

**Comment by bear** [comment]
What version of OS X is this?

**Comment by chrisfosterelli** [comment]
@bear 10.10.5

**Comment by bear** [comment]
thanks, we are looking into this

**Comment by evilpacket** [comment]
@chrisfosterelli if you try and `npm i requiresafe -g` outside of a directory with a .git in it does it work?

**Comment by chrisfosterelli** [comment]
@evilpacket Nope, has the same behaviour

**Comment by evilpacket** [comment]
@chrisfosterelli Thanks. We'll track down the issue. We have a bit of an idea.

**Comment by chrisfosterelli** [comment]
Not sure if this is relevant, but it appears to be trying to write to `/usr/local/.git/pre-commit` but `/usr/local/.git` is not root-writable so that might explain the permission issue.

```bash
➜  workspace  ls -al /usr/local/
total 96
drwxrwxr-x  23 root             admin    782 21 Jul 10:24 .
drwxr-xr-x@ 12 root             wheel    408  6 May 22:19 ..
drwxr-xr-x  14 chrisfosterelli  admin    476  6 May 22:19 .git
-rw-r--r--   1 chrisfosterelli  admin    301  6 May 22:19 .gitignore
-rw-r--r--   1 chrisfosterelli  admin    261  6 May 22:19 .yardopts
-rw-r--r--   1 chrisfosterelli  admin   3160  6 May 22:19 CODEOFCONDUCT.md
-rw-r--r--   1 chrisfosterelli  admin   1103  6 May 22:19 CONTRIBUTING.md
drwxr-xr-x  23 chrisfosterelli  admin    782 26 Sep 12:52 Cellar
-rw-r--r--   1 chrisfosterelli  admin   1241  6 May 22:19 LICENSE.txt
drwxr-xr-x  11 chrisfosterelli  admin    374 14 Aug 10:55 Library
-rw-r--r--   1 chrisfosterelli  admin   2064  6 May 22:19 README.md
-rw-r--r--   1 chrisfosterelli  admin  23781  6 May 22:19 SUPPORTERS.md
drwxr-xr-x  89 chrisfosterelli  admin   3026 26 Sep 12:52 bin
drwxr-xr-x   5 chrisfosterelli  admin    170 24 Jun 14:36 etc
drwxr-xr-x   6 chrisfosterelli  admin    204 22 Jun 10:39 foreman
drwxr-xr-x  22 root             wheel    748 17 Feb  2015 go
drwxr-xr-x   7 65535            65535    238 22 Jun 10:39 heroku
drwxr-xr-x  27 chrisfosterelli  admin    918 24 Jun 14:36 include
drwxr-xr-x  53 chrisfosterelli  admin   1802 24 Jun 14:36 lib
drwxr-xr-x  23 chrisfosterelli  admin    782 26 Sep 12:52 opt
drwxr-xr-x  10 chrisfosterelli  admin    340 21 Jul 10:24 sbin
drwxr-xr-x  14 chrisfosterelli  admin    476 26 Aug 15:34 share
drwxr-xr-x   3 chrisfosterelli  admin    102 24 Jun 14:36 var
➜  workspace  
```

If I go into that directory and run `git remote -v` it looks like the `.git` directory is being initialized and setup by homebrew.

**Comment by evilpacket** [comment]
@chrisfosterelli just published 2.0.3 which I think will resolve the issue

**Comment by chrisfosterelli** [comment]
@evilpacket Works great now! Thanks guys!
